docs/UserGuide.adoc

```diff
@@ -49,48 +53,77 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
-- Examples:
-* `inventory remove i/Egg` +
-Remove all eggs from your inventory.
-* `inventory remove i/Butter q/200g` +
-Removes 200g of butter from your inventory.
+- Format: `cookbook add n/NAME [i/INGREDIENT] [q/QUANTITY] [x/STEP_INDEX] [s/STEP] [t/TAG]`
+- Example:
+* `cookbook add n/Bacon Carbonara` +
+    Adds a new empty recipe with the name 'Bacon Carbonara'.
+* `cookbook add n/Chicken Ham Sandwich i/Chicken Ham i/White Bread q/2 slices
+      x/1 s/Toast the bread x/2 s/Grill the ham. t/sandwich t/easy` +
+    Adds a new recipe with the name 'Chicken Ham Sandwich' and the following recipe details. +
+    Tags: sandwich, easy +
+    Ingredients: Ham, Bread (2 slices) +
+    Preparation Steps: +
+    1. Toast the ham +
+    2. Grill the bread
 
-==== View inventory of ingredients
-This command allows you to view your current list of ingredients. You may add in an
-additional argument for a particular ingredient you are looking for in your database.
-If no additional arguments for the ingredient is supplied, it will display the entire inventory.
+==== Remove a recipe from the cookbook
+This command allows you to remove a recipe of the specified index from the cookbook.
 
-- Format: `inventory view`
+- Format: `cookbook remove INDEX`
+- Example:
+* `cookbook remove 1` +
+    Removes recipe 1 from the cookbook.
+
+==== Search recipes by keyword
+This command allows you to search for a recipe by a keyword, and the desired recipe can be viewed using the command `cookbook view recipe`.
+
+- Format: `cookbook search w/KEYWORD`
+- Example:
+* `cookbook search w/Carbonara` +
+    Searches the cookbook for recipes with names matching the keyword 'Carbonara'.
+
+==== Search recipes by tag
+This command allows you to search for recipes by tags, and the desired recipe can be viewed using the command `cookbook view recipe`.
+
+- Format: `cookbook search t/TAG…`
 - Examples:
-* `inventory view` +
-Displays the entire inventory
+* `cookbook search t/Easy` +
+    Searches the cookbook for recipes with tags matching 'Easy'.
+* `cookbook search t/Pasta t/Cream t/Easy` +
+    Searches the cookbook for recipes with tags matching 'Pasta', 'Cream', and 'Easy'.
+
+==== Search recipes by ingredients owned
+This command allows you to see what you can cook with your current inventory of ingredients by searching for recipes by the percentage of required ingredients owned, and the desired recipe can be viewed using the command `cookbook view recipe`.
+
+- Format: `cookbook search inventory`
+- Example:
+* `cookbook search inventory` +
+    Searches the cookbook for recipes whose ingredients are owned by you.
+
 
-=== Recipe commands
+=== Recipe Commands
 Recipe commands are commands that change a particular recipe, which you can indicate using the index of the recipe.
 Some things you can do are to add and remove ingredients from a recipe's ingredient list.
-You can also write instructions for your recipes.
+You can also write preparation steps for your recipes.
```

Comment:
Should we call it preparations steps or cooking steps or how? The headings below say "cooking steps".

Reply:
Will standardise to 'preparation step(s)' for consistency. Thank you for pointing out.